README.md

```diff
@@ -1 +1,18 @@
-# dcf-dataservice
\ No newline at end of file
+# dcf-dataservice
+Copy data buckets from GDC to AWS and Google buckets. The jobs need to be run inside a setup kubeneste cluster.
+
+GDC has two different data storages: Data center and AWS backup bucket which are synchronized continuously with each other. Every one or two months, GDC releases new/updated/deleted data in manifest files, the goal is to deploy two separated jobs which copy the data to AWS, GOOGLE cloud respectively.
+
+## GDC AWS bucket to DCF AWS buckets
+Simplely using AWS API to sync in-AWS buckets. To boost up the performance, we deploy multiple-thread aws s3 copying.
+
+## GDC data center to DCF GOOGLE buckets
+Deploy a data-flow job that copies data from GDC to GCP buckets. The `copy` transform of the data-flow pipeline streams objects from GDC data center to GOOGLE buckets.
+
+## How to run
+Following the below steps to setup
+- Create a directory in `$vpc_name/apis_configs` and named as `dcf_dataservice`.
+- Put the `creds.json`, `aws_creds_secret` and `gcloud-creds-secret` into `dcf_dataservice` folder. While `aws_creds_secret` contains AWS key `gcloud-creds-secret` contains google cloud service account.
```

Comment:
Is this correct? The google creds are a service account key in JSON format?
```suggestion
 - Put the `creds.json`, `aws_creds_secret` and `gcloud-creds-secret` into `dcf_dataservice` folder. While `aws_creds_secret` contains AWS key `gcloud-creds-secret` contains Google Cloud service account key (JSON credentials).
```

Reply:
Google returns service account credential in json format